.github/workflows/dev-integritee-restart.yml

```diff
@@ -8,6 +8,7 @@ on:
         description: Select the action
         options: 
         - stop
+        - start
```

Comment:
So we still need it?

Reply:
yes, I tried adding only restart. but if the service is not running on the server, the restart fails at "stop" action. 
thought to have "start" action as well if the service is not running already.

Reply:
I do think `systemctl  restart` can be used to start a stopped service.

You can simply ignore the error when stopping the service, either redirect the errors or something like
`sudo systemctl stop integritee.service || true` so that GHA can continue to run

Reply:
Thank you for the input. I am not aware of this option. Let me check and make the changes.

Reply:
I have made the changes. Adding `|| true` works fine. 
Thank you for the suggestion, Kai.